ros/ubuntu/Dockerfile

```
@@ -0,0 +1,83 @@
+## Provide Orocos Toolchain (RTT/OCL) and rtt_ros_integration overlay workspace on top of the
+## official ROS 2 docker images.
+
+ARG ROS_DISTRO
+ARG UBUNTU_DISTRO
+
+## *************************************************************************************************
+## Stage base: prepare build environment
+## *************************************************************************************************
+FROM ros:${ROS_DISTRO}-ros-base-${UBUNTU_DISTRO} AS base
+
+# add custom rosdep rule file and update rosdep database
+COPY prereqs.yaml /etc/ros/rosdep/
+RUN echo "yaml file:///etc/ros/rosdep/prereqs.yaml" > /etc/ros/rosdep/sources.list.d/10-custom.list
+RUN rosdep update
+
+# install python-vcstool used to clone repositories below
+RUN apt-get update \
+    && apt-get install -y python-vcstool \
```

In Ubuntu `focal`, the package `python-vcstool` doesn't exist, since python3 is the default version (`python3-vcstool`). This impedes Ubuntu `focal` ROS `noetic` to be built. Also, `git` is missing in `noetic` OSRF ([core](https://github.com/osrf/docker_images/blob/master/ros/noetic/ubuntu/focal/ros-core/Dockerfile), [base](https://github.com/osrf/docker_images/blob/master/ros/noetic/ubuntu/focal/ros-base/Dockerfile)).

```suggestion
    && if [ $(lsb_release -cs) = "focal" ] ; then VCS_PKGS="python3-vcstool git" ; else VCS_PKGS="python-vcstool" ; fi \
    && apt-get install -y ${VCS_PKGS} \
```